app/views/hyrax/base/_form_rendering.html.erb

```diff
@@ -0,0 +1,15 @@
+  <div class="row">
+    <div class="col-md-12">
+      <fieldset id="representative-media">
+        <legend>
+          Rendering
+        </legend>
+        <p>Select file(s) to be offered as as downloads for every image in Universal Viewer, for example a PDF of the whole <%= f.object.human_readable_type %>.</p>
+        <%= f.select :rendering_ids,
+                     @form.select_files,
```

Comment:
Use `f.object` rather than `@form`. I think that'll fix your `TODO` in the test down below.